packages/jest/.eslintrc.json

```diff
@@ -4,13 +4,28 @@
   "overrides": [
     {
       "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
-      "rules": {}
+      "rules": {
+        "no-restricted-imports": [
+          "error",
+          {
+            "name": "chalk",
+            "message": "Please use `picocolors` in place of `chalk` for rendering terminal colors"
+          }
+        ]
+      }
     },
     {
       "files": ["**/*.ts"],
       "excludedFiles": ["./src/migrations/**"],
       "rules": {
-        "no-restricted-imports": ["error", "@nx/workspace"]
+        "no-restricted-imports": [
+          "error",
+          "@nx/workspace",
+          {
+            "name": "chalk",
+            "message": "Please use `picocolors` in place of `chalk` for rendering terminal colors"
+          }
+        ]
```

Comment:
Let's move `excludedFiles` and the rule definition here to the block for `"files": ["*.ts", "*.tsx", "*.js", "*.jsx"]`. So, we have the rule in a single block that includes all js/ts files except migrations.

After that, this block should have empty rules and no `excludedFiles`. We'll later clean up some boilerplate in these config files, but for now, just ensuring we don't duplicate that rule needlessly is enough.

Reply:
I looked into it and we don't need the `excludedFiles` anymore because the old migrations using `@nx/workspace` don't exist anymore.